heap.py

```diff
@@ -0,0 +1,68 @@
+from dynamic_array import DynamicArray
+class MinHeap:
+    def __init__(self):
+        self.array = DynamicArray(10)
+
+    def add(self, value):
+        self.array.append(value)
+        self.heapify()
+
+    def delete(self):
+        if self.array.is_empty():
+            return False
+        self.swap_elements(0, self.array.fetch_size()-1)
```

Comment:
After swap_elements, there's no delete operation happening, we should add the below line :
```
self.array.delete(self.array.fetch_size()-1)
```

Reply:
changed delete method